docs/08_cli.md

```diff
@@ -201,3 +201,24 @@ To reset multiple workflows, you can use batch reset command:
 ```
 ./cadence workflow reset-batch --input_file <file_of_workflows_to_reset> --reset_type <reset_type> --reason "some_reason"
 ```
+#### Recovery from bad deployment -- auto-reset workflow
+If a bad deployment let a workflow run into a wrong state, you might want to reset the workflow to the point that the bad deployment started to run. But usually it is not easy to find out all the workflows impacted, and every reset point for each workflow. In this case, auto-reset will automatically reset all the workflow by the bad deployment given a identfier. 
+
+First of all, let's familar with some concepts. Each deployment will have a identifier, we call it "**Binary Checksum**" as it is usually generated by the md5sum of a binary file. For a workflow, each binary checksum will be associated with an **auto-reset point**, which contains a **runID**, an **eventID**, and the **created_time** that binary/deployment made the first decision for the workflow.
+
+To find out which **binary checksum** of bad deployment to reset, you should be aware of at least one workflow running into a bad state. Use describe command with **--reset_points_only** option to show all the reset points:
+```
+./cadence wf desc -w <>  --reset_points_only
++----------------------------------+--------------------------------+--------------------------------------+---------+
+|         BINARY CHECKSUM          |          CREATE TIME           |                RUNID                 | EVENTID |
++----------------------------------+--------------------------------+--------------------------------------+---------+
+| c84c5afa552613a83294793f4e664a7f | 2019-05-24 10:01:00.398455019  | 2dd29ab7-2dd8-4668-83e0-89cae261cfb1 |       4 |
+| aae748fdc557a3f873adbe1dd066713f | 2019-05-24 11:01:00.067691445  | d42d21b8-2adb-4313-b069-3837d44d6ce6 |       4 |
+...
+...
+```
+
+Then use this command to tell Cadence auto-reset all workflows impacted by the bad deployment. The bad binary checksum needs to store in domain info.
```

Comment:
Cadence auto-reset -> Cadence to auto-reset

I'm not sure what this means: The bad binary checksum needs to store in domain info. What is being stored and where?

Reply:
Oh if you look at the command, it is updating the domain info. I am trying explain why. But maybe it is not necessary if it leads more confusion.

Reply:
I changed it a little bit hope it makes more sense.

Reply:
Your update does make more sense :) Thanks!